### ISSUE TITLE
manual: update/remove/fix references to artiq_corelog etc

### ISSUE THREAD

**Issue by jordens** [m-labs/artiq#1008]
since d446a3293ec2869aef4b2ec846b553975a38a01d

```
doc/manual/utilities.rst:163:   :ref: artiq.frontend.artiq_corelog.get_argparser
doc/manual/utilities.rst:164:   :prog: artiq_corelog
```

**Comment by whitequark** [comment]
Ah, I forgot that, sorry.

**Comment by whitequark** [comment]
Fixed in 0c6f6180560c.

**Comment by jordens** [comment]
d446a3293ec2869aef4b2ec846b553975a38a01d also removes `aqctl_corelog`. That's a controller which is used throughout the examples. That was probably unintentional and needs to be reverted or fixed.

```
doc/manual/ndsp_reference.rst:8:   :ref: artiq.frontend.aqctl_corelog.get_argparser
doc/manual/ndsp_reference.rst:9:   :prog: aqctl_corelog
setup.py:37:    "aqctl_corelog = artiq.frontend.aqctl_corelog:main",
RELEASE_NOTES.rst:121:  RTIO collisions. A new controller ``aqctl_corelog`` must be running to forward
artiq/examples/sayma_drtio/device_db.py:14:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/master/device_db.py:19:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/kasli_basic/device_db_sysu.py:14:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/kasli_basic/device_db_mitll.py:14:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/kasli_basic/device_db_tester.py:14:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/kasli_basic/device_db_ustc.py:14:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/kc705_nist_clock/device_db.py:18:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/sayma_standalone/device_db.py:14:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/kasli_opticlock/device_db.py:17:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
artiq/examples/kasli_suservo/device_db.py:14:        "command": "aqctl_corelog -p {port} --bind {bind} " + core_addr
```
